xmind/tests/test_sheet_element.py

```diff
@@ -0,0 +1,414 @@
+        _owner_workbook = Mock()
+        _get_owner_workbook = self._init_patch_with_name('_get_owner_workbook', 'xmind.core.sheet.SheetElement.getOwnerWorkbook', return_value=_owner_workbook)
+        _obj_topic_element = Mock()
+        _topic_element = self._init_patch_with_name('_topic_element', 'xmind.core.sheet.TopicElement', return_value=_obj_topic_element)
+        _append_child = self._init_patch_with_name('_append_child', 'xmind.core.sheet.SheetElement.appendChild')
+
+        _obj = SheetElement()
+        self._assert_init_methods()
+        self._remove_patched_function("_get_root_topic")
+        self.assertEqual(_obj._get_root_topic(), _obj_topic_element)
+
+        self.assertEqual(_get_child_nodes_by_tag_name.call_count, 1)
+        _get_child_nodes_by_tag_name.assert_called_with(TAG_TOPIC)
+
+        self.assertEqual(_get_owner_workbook.call_count, 1)
+
+        self.assertEqual(_topic_element.call_count, 1)
+        _topic_element.assert_called_with(ownerWorkbook=_owner_workbook)
+
+        self.assertEqual(_append_child.call_count, 1)
+        _append_child.assert_called_with(_obj_topic_element)
+
+    def test_get_root_topic_with_topic(self):
+        _get_child_nodes_by_tag_name = self._init_patch_with_name('_get_child_nodes_by_tag_name', 'xmind.core.sheet.SheetElement.getChildNodesByTagName', return_value=[1])
+        _owner_workbook = Mock()
+        _get_owner_workbook = self._init_patch_with_name('_get_owner_workbook', 'xmind.core.sheet.SheetElement.getOwnerWorkbook', return_value=_owner_workbook)
+        _obj_topic_element = Mock()
+        _topic_element = self._init_patch_with_name('_topic_element', 'xmind.core.sheet.TopicElement', return_value=_obj_topic_element)
+
+        _obj = SheetElement()
+        self._assert_init_methods()
+        self._remove_patched_function("_get_root_topic")
+
+        self.assertEqual(_obj._get_root_topic(), _obj_topic_element)
+
+        self.assertEqual(_get_child_nodes_by_tag_name.call_count, 1)
+        _get_child_nodes_by_tag_name.assert_called_with(TAG_TOPIC)
+
+        self.assertEqual(_get_owner_workbook.call_count, 1)
+
+        self.assertEqual(_topic_element.call_count, 1)
+        _topic_element.assert_called_with(1, _owner_workbook)
+
+    def test_create_relationship_without_title(self):
+        _rel = Mock()
+        _rel.setEnd1ID.return_value = None
+        _rel.setEnd2ID.return_value = None
+        _rel.setTitle.return_value = None
+        _owner_workbook = Mock()
+        _get_owner_workbook = self._init_patch_with_name('_get_owner_workbook', 'xmind.core.sheet.SheetElement.getOwnerWorkbook', return_value=_owner_workbook)
+        _relationship_element = self._init_patch_with_name('_relationship_element', 'xmind.core.sheet.RelationshipElement', return_value=_rel)
+
+        _obj = SheetElement()
+        self._assert_init_methods()
```

Comment:
move check for init methods to the end of test, it will make easier to read the test